<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BySMOContingentSetupDialog</class>
 <widget class="QDialog" name="BySMOContingentSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblPolicyDate">
     <property name="text">
      <string>Дата</string>
     </property>
     <property name="buddy">
      <cstring>edtPolicyDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtPolicyDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbPolicyKind"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPolicyKind">
     <property name="text">
      <string>Вид полиса</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPolicyType">
     <property name="text">
      <string>Тип полиса</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbPolicyType"/>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblRazrNas">
     <property name="text">
      <string>В разрезе населения</string>
     </property>
     <property name="buddy">
      <cstring>cmbRazrNas</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="cmbRazrNas">
     <item>
      <property name="text">
       <string>Все</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Краевые</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Инокраевые</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtPolicyDate</tabstop>
  <tabstop>cmbPolicyType</tabstop>
  <tabstop>cmbPolicyKind</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BySMOContingentSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BySMOContingentSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
